templates/controllers/wizard/fileUpload/form/submissionFileMetadataForm.tpl

```diff
@@ -29,19 +29,6 @@
 		{/fbvFormSection}
 	{/fbvFormArea}
 
-	{* Read-only meta-data *}
```

Comment:
It's my understanding that this data is never editable in this interface. If it is exposed to indexing in some way, what's removed here doesn't help in tailoring that at all.

Are you just wanting the information to be visible so an editor knows they need to upload a resized or converted copy?